sdk/textanalytics/Azure.AI.TextAnalytics.Legacy.Shared/src/Azure.AI.TextAnalytics.Legacy.csproj

```diff
@@ -0,0 +1,34 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
To make sure that things play nice with the generator, I used a standard project here, rather than using a "shared code" type project.   This allows us to also test compilation of the generated surface independently.   

The only downstream impact is that we'll need to do source linking via glob pattern rather than having a single ready-made package of code to import.

Reply:
There won't be a shared class between them - at least not in the sense of partials or true code sharing.   There may be custom types that are used by both, such as the `Transformer`, but there should not be anything that is generated that crosses between the two layers.   Editing things shouldn't require any special steps.